### DISCUSSION TITLE
Unable to publish my action to the marketplace

### DISCUSSION THREAD

**Discussion by xt0rted** [community/community#25879] [Actions]
<p>I had published an action to the marketplace during the v1 preview, but there were some issues with it and I didn’t have the time to fix them so I unpublished it. I’ve since fixed the issues and updated it to v2 but when I went to publish it I got an error saying the name must be unique. There’s no action in the marketplace with this name, just this one before I unpublished it. Is there a way to publish this without changing the name? This is the action <a href="https://github.com/xt0rted/block-autosquash-commits-action" rel="nofollow noopener">https://github.com/xt0rted/block-autosquash-commits-action</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/99fafbe40f1453c398752f0db32c8d4e110dd56b.jpeg" data-download-href="/uploads/short-url/lYaMAbTxqTiOvm6OL1BV36XP0wj.jpg?dl=1" title="EFFsnoSXYAY4qf1.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/831974/181093808-ffa1d39b-e623-4548-9bf2-87cf6a664174.jpeg" alt="EFFsnoSXYAY4qf1.jpg" data-base62-sha1="lYaMAbTxqTiOvm6OL1BV36XP0wj" width="629" height="499" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/99fafbe40f1453c398752f0db32c8d4e110dd56b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">EFFsnoSXYAY4qf1.jpg</span><span class="informations">1151×914 49.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/xt0rted">@xt0rted</a>,</p>
<p>Thank you for being here! This is a known issue and our Engineering team is working on it. I’ll be user to keep this post updates as soon as we get news.</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>We just shipped a fix for this. Could you give it another try?</p>
<p>Sorry about that.</p>

**Comment by xt0rted** [comment]
<p>Thanks, it works. I was able to republish to the marketplace.</p>
